src/server/server.js

```diff
@@ -25,6 +25,19 @@ app.get("/weather/:lat/:lng", (req, res) => {
   });
 });
 
+app.get("/stock/:nse", (nse) => {
+  const {nse} = req.params;
+  request({
+    url: getStockUrl(nse),
+    method: 'GET',
+    headers: {
+      'Content-Type':'application/x-www-form-urlencoded; charset=UTF-8'
+    }
+  }, (error, response, body) => {
+    res.send(JSON.parse(body));
```

Comment:
I realize that this is redundant since we're doing `JSON.parse(body)` when we grab it again in the component later... I should change that code in weather when I get the chance. `res.send(body)` should be fine.